packages/grid/src/BaseCell.css

```diff
@@ -76,5 +75,5 @@
   border-top: 0 solid transparent;
   border-left: var(--grid-editableCell-cornerTag-size) solid var(--grid-cursor-borderColor);
   border-bottom: 8px solid transparent;
-  z-index: 10;
+  /*z-index: 10;*/
```

Comment:
Needed if commented out or can we delete?

Reply:
I was searching and deleting these. One left. Will remove in the next pr. Thanks